test/e2e/performanceprofile/functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -168,6 +172,43 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 		})
 	})
 
+	It("[test_id:XXXX1] Verify PPC fails when offlined cpu count is more than available cpus", func() {
+		ppcArgs := []string{
+			fmt.Sprintf("--reserved-cpu-count=%d", 1),
+			fmt.Sprintf("--offlined-cpu-count=%d", 100),
+			"--rt-kernel=false",
+		}
+		cmdArgs := append(defaultArgs, ppcArgs...)
+		_, errData, _ := testutils.ExecAndLogCommandWithStderr(ppcPath, cmdArgs...)
+		ppcErrorString := errorStringParser(errData)
+		Expect(ppcErrorString).To(ContainSubstring("please specify the offlined CPU count in the range"))
+	})
+
+	It("[test_id:XXXX2] Verify PPC fails when offlined cpu count is negative", func() {
+		ppcArgs := []string{
+			fmt.Sprintf("--reserved-cpu-count=%d", 1),
+			fmt.Sprintf("--offlined-cpu-count=%d", -1),
+			"--rt-kernel=false",
+		}
+		cmdArgs := append(defaultArgs, ppcArgs...)
+		_, errData, _ := testutils.ExecAndLogCommandWithStderr(ppcPath, cmdArgs...)
+		ppcErrorString := errorStringParser(errData)
+		Expect(ppcErrorString).To(ContainSubstring("please specify the offlined CPU count in the range"))
+	})
+
+	It("[test_id:XXXX3] Verify PPC fails when the sum of offlined cpu count and reserved cpu count is more than available cpus", func() {
+		ppcArgs := []string{
+			fmt.Sprintf("--offlined-cpu-count=%d", 50),
+			fmt.Sprintf("--reserved-cpu-count=%d", 50),
+			"--rt-kernel=false",
```

Comment:
do we want e2e tests for:

- offline and reserved sets overlapping
- all the above with rt-kernel=true?

Reply:
I can add them but `rt-kernel` is not checked to calculate any of the cpuSets.

Reply:
ok, then no point. We're good as we are now.

Reply:
ok. the parameter is irrelevant but it is mandatory.